### PR TITLE
misc fixes & tidy

### LINKED ISSUE #3: multiple FROM
[AttackDeploy/Dockerfile](https://github.com/ZephrFish/DockerAttack/blob/master/AttackDeploy/Dockerfile) has two `FROM` statements, the first is completely lost since there's no later reference to it (no `as <name>` or subsequent `COPY --from`) [[1]] - did you intend to build two completely independent and different images?

[1]: https://docs.docker.com/engine/reference/builder/#from

### PR DESCRIPTION
**Pull request by casperdcl**
- fix #3 in a way that was probably intended
- fix build
- remove `dist-upgrade` (https://github.com/sameersbn/docker-redmine/issues/49#issuecomment-59355063)
- added `docker-compose`

### CODE REVIEW

File: AttackDeploy/Dockerfile [hunk 1]
```diff
@@ -1,12 +1,14 @@
 # Set up the basics 
 # Leverage Kali Base Image
-FROM kalilinux/kali-linux-docker
+FROM kalilinux/kali-linux-docker:latest
```
Comment (n4ncy) [inline AttackDeploy/Dockerfile:3]: I wouldn't add the latest tag, add a more specific version. Adding latest could cause problems if a broken kali image is put into production.

**Review by n4ncy — COMMENTED** [review]
Also would be a good idea for you to remove all the unnecessary `apt-get update`

File: AttackDeploy/Dockerfile (hunk 1 above)
Comment (casperdcl) [reply AttackDeploy/Dockerfile:3]: There appears to be only 3 tags to choose from https://hub.docker.com/r/kalilinux/kali-linux-docker/tags/ - `latest`, `kali-rolling`, and `Rolling`.

Comment (n4ncy) [reply AttackDeploy/Dockerfile:3]: @casperdcl Maybe a good idea to make a new repo just specifically for it. Although don't want to over kill. I guess leave as is the now.

**Review by n4ncy — APPROVED** [review]

**Pull request merged**